tests/migration/postcopy.go

```diff
@@ -257,22 +244,25 @@ var _ = SIGMigrationDescribe("VM Post Copy Live Migration", func() {
 						}, 120*time.Second, 1*time.Second).Should(Succeed(), "Virt handler should come online")
 					}
 					It("should make sure that VM restarts after failure", func() {
-
 						By("creating a large VM with RunStrategyRerunOnFailure")
-						vm := createLargeVirtualMachine()
+						vmi := libvmifact.NewFedora(
+							libnet.WithMasqueradeNetworking(),
+							libvmi.WithResourceMemory("3Gi"),
+							libvmi.WithRng(),
+							libvmi.WithNamespace(testsuite.NamespacePrivileged),
```

Comment:
in a follow-up PR, please ensure that the VM runs in a normal namespace. Only the killer pod should be privileged

Reply:
Let's try it :)